src/mongocxx/test/client_side_encryption.cpp

```diff
@@ -2390,4 +2393,261 @@ TEST_CASE("Explicit Encryption", "[client_side_encryption]") {
     }
 }
 
+TEST_CASE("Unique Index on keyAltNames", "[client_side_encryption]") {
+    instance::current();
+
+    if (!mongocxx::test_util::should_run_client_side_encryption_test()) {
+        std::cerr << "Skipping Unique Index on keyAltNames prose tests" << std::endl;
+        return;
+    }
+
+    if (!test_util::newer_than(uri{}, "4.2")) {
+        std::cerr << "Unique Index on keyAltNames tests require MongoDB server 4.2+." << std::endl;
```

Comment:
```suggestion
        WARN ("Skipping - requires MongoDB server 4.2+");
```
Use Catch2 `WARN` instead of `std::cerr`. Context (test section) will be automatically included in test output.